user_manual/pid_gen/pid_gen.cpp

```diff
@@ -0,0 +1,237 @@
+/*
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Library General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * pid_gen.cpp
+ * Generate the markdown tables from the supported PIDs.
+ * Copyright (C) 2016 Simon Newton
+ */
+
+#include <ola/Constants.h>
+#include <ola/base/Flags.h>
+#include <ola/base/Init.h>
+#include <ola/base/SysExits.h>
+#include <ola/file/Util.h>
+#include <ola/network/NetworkUtils.h>
+#include <ola/rdm/PidStoreHelper.h>
+#include <ola/rdm/RDMCommand.h>
+#include <ola/rdm/RDMCommandSerializer.h>
+#include <ola/rdm/RDMEnums.h>
+#include <ola/rdm/UID.h>
+
+#include <string.h>
+#include <stdlib.h>
+#include <algorithm>
+#include <fstream>
+#include <memory>
+#include <vector>
+
+#include "Array.h"
+#include "dimmer_model.h"
+#include "led_model.h"
+#include "moving_light.h"
+#include "network_model.h"
+#include "proxy_model.h"
+#include "sensor_model.h"
+#include "rdm.h"
+#include "rdm_buffer.h"
+#include "rdm_responder.h"
+
+using ola::rdm::PidDescriptor;
+using ola::rdm::PidStoreHelper;
+using ola::rdm::UID;
+using std::cerr;
+using std::cout;
+using std::endl;
+using std::setw;
+using std::string;
+using std::unique_ptr;
+using std::vector;
+
+const uint8_t TEST_UID[] = {0x7a, 0x70, 0, 0, 0, 1};
```

Comment:
Not that it really matters, but any reason not to use the Ja-Rule non-MAC UID 7a70:fffffe00?